src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,76 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public class RandomCharsTable {
```

Comment:
Opt: In general, it is good. Char aggregation by strategy is difficult to understand, if possible, simplify. And please follow Code conventions on blank lines/empty spaces